proposals/incubation/credebl.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: default
+title: CREDEBL
+parent: Incubation
+grand_parent: Project Proposals
+---
+
+# CREDEBL Project Proposal
+
+## CREDEBL
+**CREDEBL** is a Decentralized Identity & Digital Credentialing Platform. It is recognized as a Digital Public Good (DPG) by the Digital Public Good Alliance (DPGA).
+
+# Sponsor(s)
+* _Ajay Jadhav (ajay@ayanworks.com_)
+* _Kalyan Kulkarni (kalyan@ayanworks.com)_
+* _Deepak Ghadge (deepak.ghadge@ayanworks.com)_
+* _Ankita Patidar (ankita.patidar@ayanworks.com)_
+* _Amit Padmani (amit.padmani@ayanworks.com)_
+* _Prasad Kalamkar (prasad.kalamkar@ayanworks.com)_
+
+# Abstract
+CREDEBL is an industry-leading Verifiable Credential & Decentralised Identity management platform which helps in building scalable, secure and user-centric SSI solutions. The multi-tenant platform offers straightforward interactions, enabling rapid configuration & deployment of any DID/VC solution.
+
+# Context
+CREDEBL Platform is conceptualized & built by AYANWORKS. After building & delivering several decentralized identity systems, AYANWORKS team felt a need of a system which acts as a re-usable core SSI platform and support in building multiple sector-specific use cases of Decentralized Identity & Verifiable Credentials implementation.
+
+Hence, CREDEBL is built with support for multi-tenancy while being agnostic to underlying ledgers/VDRs, DID methods, and Verifiable Credential formats.
```

Comment:
What does multi-tenancy mean in this context?
Could you spell out acronyms such as VDR for people not familiar with this space.

Reply:
In a Digital Trust Ecosystem such as National Digital ID, Academic Credentials or Digital Travel Credentials, multi-tenancy means a single instance of CREDEBL Platform can be used to onboard multiple tenants (e.g. Credential Issuer and/or Verifier organizations).

VDR = Verifiable Data Registry / Blockchain / Distributed Ledger, which is generally used to register DIDs of Issuers & Verifiers. Ref: [W3C DID Spec](https://www.w3.org/TR/did-core/#dfn-verifiable-data-registry)